_targets.R

```diff
@@ -6,10 +6,44 @@ source("3_visualize/src/plot_timeseries.R")
 options(tidyverse.quiet = TRUE)
 tar_option_set(packages = c("tidyverse", "dataRetrieval")) # Loading tidyverse because we need dplyr, ggplot2, readr, stringr, and purrr
 
+p_width <- 12
+p_height <- 7
+p_units <- "in"
```

Comment:
I like that you've pulled out these parameters and defined them at the top of the `_targets.R` file. Could you also pull out the `parameterCd`, `startDate` and `endDate` parameters used in the `download_nwis_site_data()` function?

Reply:
That way anyone who uses the pipeline could easily change what type of data is downloaded or the period for which data is downloaded.

Reply:
Yes!